modules/cert-manager-acme-dns01-explicit-azure.adoc

```diff
@@ -0,0 +1,163 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-acme-dns01-explicit-azure_{context}"]
+= Configuring an ACME issuer to solve DNS-01 challenges by using explicit credentials for Microsoft Azure DNS
+
+You can use {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using explicit credentials on Microsoft Azure. This procedure uses _Let's Encrypt_ as the ACME CA server and shows how to solve DNS-01 challenges with Azure DNS.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
```

Comment:
If one section needs `cluster-admin`, why other sections do not need it?
Actually, if one section just creating issuer, certificate, that section needs not cluster-admin.
If one section needs change certmanager/cluster, or patch subscription, or create namespace, or create credentialsrequest, et al, then it needs cluster-admin.
BTW suggest replace creating namespace with `oc new-project`, oc new-project needs not cluster-admin.